charts/cluster-autoscaler/values.yaml

```diff
@@ -125,6 +125,9 @@ dnsPolicy: ClusterFirst
 # See: https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/expander/priority/readme.md
 expanderPriorities: {}
 
+# priorityConfigMapAnnotations -- Annotations to add to `cluster-autoscaler-priority-expander` ConfigMap.
+priorityConfigMapAnnotations: {}
+
```

Comment:
Can you add a commented out example for users here please?

Reply:
added in ba47d03